jobrunner/create_or_update_jobs.py

```diff
@@ -188,8 +188,8 @@ def recursively_add_jobs(job_request, project, action, force_run_actions):
     job = Job(
         job_request_id=job_request.id,
         state=State.PENDING,
-        repo_url=action_spec.repo_url if action_spec.repo_url else job_request.repo_url,
-        commit=action_spec.commit_sha if action_spec.commit_sha else job_request.commit,
+        repo_url=job_request.repo_url,
+        commit=job_request.commit,
```

Comment:
I don't quite get how this works now. Don't we still need to pass the repo_url and commit through from the action?

It might be worth adding a reusable action step to the mammoth integration test just to make sure all the parts work together (which I think would have caught this). I agree it's a slightly intimidating giant of a test though, so happy to help out there.

Reply:
After squashing commits and tidying up, there is very little change to *jobrunner/create_or_update_jobs.py*. I think it would be a good idea to close this PR, force pushing, and opening a new PR. After a -- hopefully quick -- review, then I will add to the integration test. How does that sound to you?

Reply:
Sounds good. Just include a link back to this PR in the new PR description for future github archaeologists